CMakeLists.txt

```diff
@@ -115,12 +118,14 @@ set(valhalla_data_tools valhalla_build_statistics valhalla_ways_to_edges valhall
 ## Valhalla services
 set(valhalla_services	valhalla_service valhalla_loki_worker	valhalla_odin_worker valhalla_thor_worker)
 
-foreach(program ${valhalla_programs})
-  get_source_path(path ${program})
-  add_executable(${program} ${path})
-  target_link_libraries(${program} Boost::program_options valhalla Threads::Threads)
-  install(TARGETS ${program} DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT runtime)
-endforeach()
+if(ENABLE_DATA_TOOLS)
```

Comment:
added a new thing to avoid building these binaries if you dont need them (this removes some boost::programs_options dependencies)

Reply:
Yesterday I was thinking we should do this. Great!